src-tauri/src/buffer.rs

```diff
@@ -0,0 +1,19 @@
+ let poppedup_tabs = browser.get_tabs().lock().unwrap();
+    for tabbinger in poppedup_tabs.iter() {
+        println!("Tabbinger URL: {}", tabbinger.get_url());
+        if (tabbinger.get_url().contains("/simulate/popup")) {
```

Comment:
Is there a better way to identify the popup? This check is for the simulated app where the URL is not dynamic. However in the actual web app, the URL will change a couple of times internally. Is it possible to identify the popup by content instead of URL?

Reply:
Hi! 

Great comment and very good points. Unfortunately, I have to deeply apologize for having the buffer.rs in my pull request. It is **not** implemented in the actual running code base and it was used to buffer store my original headless_chrome implementation.